error with Enhanced Inventories

tested in regrowth, clicking `u` on dead wood planks
simply making the method public would fix this
```
[Client thread/ERROR] [NotEnoughItems]: Error while looking up usage recipe
[Client thread/ERROR] [NotEnoughItems]: inputId: item
[Client thread/ERROR] [NotEnoughItems]: ingredients: [1xitem.deadPlank@0]
java.util.concurrent.ExecutionException: java.lang.IllegalAccessError
	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1006)
	at codechicken.nei.recipe.RecipeHandlerQuery.getHandlersWithRecipes(RecipeHandlerQuery.java:88)
	at codechicken.nei.recipe.RecipeHandlerQuery.getRecipeHandlersParallel(RecipeHandlerQuery.java:58)
	at codechicken.nei.recipe.RecipeHandlerQuery.runWithProfiling(RecipeHandlerQuery.java:39)
	at codechicken.nei.recipe.GuiUsageRecipe.getUsageHandlers(GuiUsageRecipe.java:55)
	at codechicken.nei.recipe.GuiUsageRecipe.openRecipeGui(GuiUsageRecipe.java:25)
	at codechicken.nei.api.ShortcutInputHandler.handleMouseClick(ShortcutInputHandler.java:117)
	at codechicken.nei.recipe.RecipeItemInputHandler.mouseClicked(RecipeItemInputHandler.java:26)
	at codechicken.nei.guihook.GuiContainerManager.mouseClicked(GuiContainerManager.java:448)
	at net.minecraft.client.gui.inventory.GuiContainer.func_73864_a(GuiContainer.java)
	at codechicken.nei.recipe.GuiRecipe.func_73864_a(GuiRecipe.java:697)
	at net.minecraft.client.gui.GuiScreen.func_146274_d(GuiScreen.java:296)
	at net.minecraft.client.gui.inventory.GuiContainer.func_146274_d(GuiContainer.java)
	at net.minecraft.client.gui.GuiScreen.func_146269_k(GuiScreen.java:268)
	at net.minecraft.client.Minecraft.func_71407_l(Minecraft.java:1640)
	at net.minecraft.client.Minecraft.func_71411_J(Minecraft.java:973)
	at net.minecraft.client.Minecraft.func_99999_d(Minecraft.java:3006)
	at net.minecraft.client.main.Main.main(SourceFile:148)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.minecraft.launchwrapper.Launch.launch(Launch.java:135)
	at net.minecraft.launchwrapper.Launch.main(Launch.java:28)
	at org.prismlauncher.launcher.impl.StandardLauncher.launch(StandardLauncher.java:87)
	at org.prismlauncher.EntryPoint.listen(EntryPoint.java:130)
	at org.prismlauncher.EntryPoint.main(EntryPoint.java:70)
Caused by: java.lang.IllegalAccessError
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.util.concurrent.ForkJoinTask.getThrowableException(ForkJoinTask.java:598)
	at java.util.concurrent.ForkJoinTask.get(ForkJoinTask.java:1005)
	... 26 more
Caused by: java.lang.IllegalAccessError: tried to access method codechicken.nei.recipe.TemplateRecipeHandler$CachedRecipe.randomRenderPermutation(Lcodechicken/nei/PositionedStack;J)V from class yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe$$anonfun$getCycledIngredients$2
	at yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe$$anonfun$getCycledIngredients$2.apply$mcVI$sp(EIShapedHandlerBasic.scala:92)
	at yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe$$anonfun$getCycledIngredients$2.apply(EIShapedHandlerBasic.scala:91)
	at yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe$$anonfun$getCycledIngredients$2.apply(EIShapedHandlerBasic.scala:91)
	at scala.collection.TraversableLike$WithFilter$$anonfun$foreach$1.apply(TraversableLike.scala:778)
	at scala.collection.immutable.Range.foreach(Range.scala:160)
	at scala.collection.TraversableLike$WithFilter.foreach(TraversableLike.scala:777)
	at yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe.getCycledIngredients(EIShapedHandlerBasic.scala:91)
	at yuuto.enhancedinventories.compat.nei.EIShapedHandlerBasic$CachedEIShapedRecipe.getIngredients(EIShapedHandlerBasic.scala:88)
	at codechicken.nei.recipe.TemplateRecipeHandler.getIngredientStacks(TemplateRecipeHandler.java:664)
	at codechicken.nei.recipe.SearchRecipeHandler.mathRecipe(SearchRecipeHandler.java:42)
	at codechicken.nei.recipe.SearchRecipeHandler.findFirst(SearchRecipeHandler.java:60)
	at codechicken.nei.recipe.RecipeHandlerQuery.lambda$null$5(RecipeHandlerQuery.java:87)
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:174)
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193)
	at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1384)
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482)
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472)
	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:747)
	at java.util.stream.ReduceOps$ReduceTask.doLeaf(ReduceOps.java:721)
	at java.util.stream.AbstractTask.compute(AbstractTask.java:327)
	at java.util.concurrent.CountedCompleter.exec(CountedCompleter.java:731)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool.helpComplete(ForkJoinPool.java:1870)
	at java.util.concurrent.ForkJoinPool.awaitJoin(ForkJoinPool.java:2045)
	at java.util.concurrent.ForkJoinTask.doInvoke(ForkJoinTask.java:404)
	at java.util.concurrent.ForkJoinTask.invoke(ForkJoinTask.java:734)
	at java.util.stream.ReduceOps$ReduceOp.evaluateParallel(ReduceOps.java:714)
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:233)
	at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:566)
	at codechicken.nei.recipe.RecipeHandlerQuery.lambda$getHandlersWithRecipes$6(RecipeHandlerQuery.java:88)
	at java.util.concurrent.ForkJoinTask$AdaptedCallable.exec(ForkJoinTask.java:1424)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175)
[Client thread/INFO]: [CHAT] There was error during loading recipe. Please check log for details.
```

--- a/src/main/java/codechicken/nei/recipe/TemplateRecipeHandler.java
+++ b/src/main/java/codechicken/nei/recipe/TemplateRecipeHandler.java
@@ -197,7 +197,7 @@ public abstract class TemplateRecipeHandler implements ICraftingHandler, IUsageH
             return ingredients;
         }
 
-        protected void randomRenderPermutation(List<PositionedStack> stacks, long cycle) {
+        public void randomRenderPermutation(List<PositionedStack> stacks, long cycle) {
             final ItemFilter filter = getItemFilter();
             final int randCycle = Math.abs(new Random(cycle + offset).nextInt());
 
@@ -228,7 +228,7 @@ public abstract class TemplateRecipeHandler implements ICraftingHandler, IUsageH
         }
 
         @Deprecated
-        protected void randomRenderPermutation(PositionedStack stack, long cycle) {
+        public void randomRenderPermutation(PositionedStack stack, long cycle) {
             Random rand = new Random(cycle + offset);
             stack.setPermutationToRender((int) (Math.abs(rand.nextInt()) % stack.items.length));
         }

